server/controllers/lists/by_entities.js

```diff
@@ -0,0 +1,39 @@
+const _ = require('builders/utils')
+const selections_ = require('controllers/lists/lib/selections')
+const filterVisibleDocs = require('lib/filter_visible_docs')
+const lists_ = require('controllers/lists/lib/lists')
+const { Paginate } = require('controllers/items/lib/queries_commons')
+
+const sanitization = {
+  uris: { },
+  limit: { optional: true },
+  offset: { optional: true }
+}
+
+const controller = async ({ uris, offset, limit, reqUserId }) => {
+  const foundSelections = await selections_.byEntities(uris)
+  const listsIds = _.uniq(foundSelections.map(_.property('list')))
```

Comment:
Yep, decision is to be specific of Selection a dedicated to a List. It will be more future proofed (and easier to move back to mutualize selection in several list).
This has the advantage to be comment specific later: if i add a Les Misérables comment which belongs to a "Kids List", i dont want this comment to be visible in the Adult List which also having Les Misérables.

Reply:
fixup! add Selection as sub-List element